### DISCUSSION TITLE
Two Newbie Questions: HOw to Check Kudos, and how to enter Maintenance Mode?

### DISCUSSION THREAD

**Discussion by CallMeAl1973** [Haidra-Org/AI-Horde#264] [General]
Hello,
Sorry if these two asks are documented somewhere. If they are, I haven’t found them yet.
I’m a newbie to contributing to the horde. Currently running Kobold AI with the included Horde Bridge. I can tell from the console that everything is working, I’m generating responses and receiving kudos; but how can I check how many kudos I have over all? If there’s a UI for that somewhere, it’s not obvious where to find it.
When you want to stop contributing, the FAQ suggests that you first put the bridge into maintenance mode. How is this done?
Thanks,

**Comment by db0** [comment]
The built-in koboldAI UI doesn't contain any such functionality,m but if you use https://lite.koboldai.net or https://aqualxx.github.io/stable-ui/dashboard they provide ways to putn your workers into maintenance and see your kudos
